person.rb

```diff
@@ -9,6 +9,12 @@ def initialize(age, name: 'Unknown', parent_permission: true)
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+    rental.owner = self
   end
```

Comment:
- [ ] in order to set up a many to many relationship the Rental should be initialized here passing the date, the person and the book, so it will look something like below 
```
def add_rental(person, date) 
  Rental.new(date, self, book)
end

```

Reply:
Hello, the way my code is structured the `add_rental` method is supposed to ad a rental to `@rentals` that the person has. the rental will already have been created by whoever is calling the method `add_rental`